Libraries/Components/WebView/WebView.android.js

```diff
@@ -264,6 +269,7 @@ class WebView extends React.Component {
         onLoadingStart={this.onLoadingStart}
         onLoadingFinish={this.onLoadingFinish}
         onLoadingError={this.onLoadingError}
+        onLoadingSslError={this.onLoadingSslError}
```

Comment:
we would need to be able to distinguish the two though. should we use all the same event code and just add an `isSslError` boolean to the event in `onReceivedSslError`?